test/functionalTest/cases/2750_common_metrics/2750_incomingTransactions.test

```diff
@@ -159,6 +162,28 @@ Date: REGEX(.*)
         }
     },
     "sum": {
+        "subServs": {
```

Comment:
Note (at https://github.com/telefonicaid/fiware-orion/issues/2750) that the right one is `"subservs", not "subServs"...